modules/rostests/apitests/umkm/Exceptions.c

```diff
@@ -0,0 +1,238 @@
+/*
+ * PROJECT:     ReactOS API Tests
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Tests for exception handling
+ * COPYRIGHT:   Copyright 2024 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#include "precomp.h"
+#include <apitest.h>
+#include <windows.h>
+#include <stdio.h>
+#include <ndk/rtlfuncs.h>
+
+void
+RunHandleRaiseExceptionTest(void)
+{
+    _SEH2_TRY
+    {
+       RaiseException(0xDEADBEEF, 0, 0, NULL);
+       exit(-1);
+    }
+    _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
+    {
+        /* This should be executed */
```

Comment:
I would be of interest, for helping the reader to know what the expected behaviour is, to have such a similar comment in the other tests below.